src/main/java/filter/expression/Qualifier.java

```diff
@@ -1031,11 +1031,9 @@ private Qualifier convertMilestoneAliasQualifier(List<TurboMilestone> allMilesto
             }
         }
 
-        // clamp
-        if (currentIndex >= allMilestones.size()) {
-            currentIndex = allMilestones.size() - 1;
-        } else if (currentIndex < 0) {
-            currentIndex = 0;
+        // if out of milestone range, don't convert alias
+        if (currentIndex >= allMilestones.size() || currentIndex < 0) {
+            return this;
```

Comment:
It's not that good of an idea, but do we have any qualifier that would fail for all issues? I was thinking it might be the case that the user has such milestone (although highly unlikely) so might as well pass it as it is.